recipes/libcugraph/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+c_compiler_version:
+  - 7
+cxx_compiler_version:
+  - 7
+
```

Comment:
I don't think this is what we want to do. You can see what cupy does here: https://github.com/conda-forge/cupy-feedstock/blob/master/recipe/conda_build_config.yaml

Reply:
Thanks, I've added the missing preprocessing selectors in the next commit. Was that all that was missing, or should I also be specifying `cuda_compiler`, `cuda_compiler_version`, `docker_image`, etc.?